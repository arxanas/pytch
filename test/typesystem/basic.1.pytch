let foo = 1
show_type(foo)
