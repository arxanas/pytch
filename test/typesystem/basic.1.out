line 2: BaseTy(reason=BuiltinReason(name='int'), name='int')
